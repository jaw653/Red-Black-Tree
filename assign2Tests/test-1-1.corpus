l
r
b
z
k
v
u
f
p
l
y
z
n
h
s
g
i
d
l
r
b
z
k
v
u
f
p
l
y
z
n
h
s
g
i
d
