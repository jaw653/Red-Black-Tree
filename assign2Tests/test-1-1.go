f l
f r
f b
f z
f k
f v
f u
f f
f p
f l
f y
f z
f n
f h
f s
f g
f i
f d
s
r
