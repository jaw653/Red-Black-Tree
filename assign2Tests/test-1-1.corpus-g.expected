Frequency of l: 4
Frequency of r: 2
Frequency of b: 2
Frequency of z: 4
Frequency of k: 2
Frequency of v: 2
Frequency of u: 2
Frequency of f: 2
Frequency of p: 2
Frequency of l: 4
Frequency of y: 2
Frequency of z: 4
Frequency of n: 2
Frequency of h: 2
Frequency of s: 2
Frequency of g: 2
Frequency of i: 2
Frequency of d: 2
0: l-4(l-4)-
1: b-2(l-4)-l r-2(l-4)-r
2: k-2(b-2)-r p-2(r-2)-l z-4(r-2)-r
3: f-2(k-2)-l =n-2(p-2)-l v-2(z-4)-l
4: =d-2(f-2)-l h-2(f-2)-r u-2(v-2)-l =y-2(v-2)-r
5: =g-2(h-2)-l =i-2(h-2)-r =s-2(u-2)-l
Words/Phrases: 36
Nodes: 16
Minimum depth: 2
Maximum depth: 6
